src/trophy.ts

```diff
@@ -137,7 +137,7 @@ export class AllSuperRankTrophy extends Trophy{
 }
 export class Joined2020Trophy extends Trophy{
   constructor(score: number){
-    const rankConditions = [
+    const rankConditions = [ 
```

Comment:
This new space does not seem right

Reply:
Good catch!
@byronbytes For the text editor you use, I'd recommend installing the [EditorConfig](https://editorconfig.org/) plugin, so that [the `.editorconfig` file](https://github.com/ryo-ma/github-profile-trophy/blob/7b8a6a4100893bbf1c721e563c6d07e04c79e479/.editorconfig) will prevent formatting issues like this.